transcom/mymove
# MB-10884 Make DestinationAddressType editable when updating and creating shipments as office user

## [Jira ticket](https://dp3.atlassian.net/browse/MB-10884) for this change

## Summary

This makes is it so the GHCAPI can manipulate the DestinationAddressType value when creating and editing MTOShipments with the GHCAPI. 

## Setup to Run Your Code

<details>
<summary>💻 You will need to use terminals to test this locally.</summary>

##### Terminal 1

Run server tests locally.

```sh
make server_test
```

##### Terminal 2

Start the Go server locally.

```sh
make server_run
```

##### Terminal 3
```sh
make office_client_run
```

##### Terminal 4
```sh
make db_dev_e2e_populate
```

</details>

## Verification Steps for Author
If you log into the office app as a services counselor and look at the move with the `DATYPE` move code you should see a destinationAddressType come back in the payload like so:
<img width="1624" alt="image" src="https://user-images.githubusercontent.com/4325613/150040832-a7306efa-b2ef-42fa-bcdb-ea28cd55c00c.png">

You should also be able to do something like send a PATCH like this one: 

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/4325613/150041153-7b725e1a-bce2-49ab-829c-0735edad47cb.png">


These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.



### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
